docs/it/extend/forms.md

```diff
@@ -0,0 +1,114 @@
+# Forms and Requests
```

Comment:
For pages under extend that aren't yet translated, please don't include the English source; instead, leave it blank and add text saying that translation is still in progress.